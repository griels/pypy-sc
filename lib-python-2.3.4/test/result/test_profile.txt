command: /usr/bin/python /u/pedronis/PyPy/dist/pypy/tool/alarm.py 900 /u/pedronis/PyPy/dist/pypy/interpreter/py.py --oldstyle /u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_profile.py
run by: pedronis@ratthing-b246
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 12:07:58 2005
pypy-revision: 11549
OUTPUT TEST
see output in: /u/pedronis/PyPy/dist/lib-python-2.3.4/test/result/test_profile.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
faking <type '_sre.SRE_Pattern'>
faking <type 'function'>
FAILED: test output differs
========================== closed ==========================
execution time: 37.9071319103 seconds
exit status: 0
