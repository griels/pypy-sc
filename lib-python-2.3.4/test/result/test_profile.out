test_profile
         53 function calls in 1.000 CPU seconds

   Ordered by: standard name

   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
        1    0.000    0.000    1.000    1.000 <string>:1(?)
        0    0.000             0.000          profile:0(profiler)
        1    0.000    0.000    1.000    1.000 profile:0(testfunc())
        1    0.399    0.399    1.000    1.000 test_profile.py:21(testfunc)
        2    0.080    0.040    0.600    0.300 test_profile.py:30(helper)
        4    0.116    0.029    0.120    0.030 test_profile.py:48(helper1)
        8    0.312    0.039    0.399    0.049 test_profile.py:56(helper2)
        8    0.063    0.007    0.079    0.009 test_profile.py:66(subhelper)
       28    0.027    0.000    0.027    0.000 test_profile.py:78(__getattr__)


